src/openforms/pre_requests/clients.py

```diff
@@ -1,4 +1,6 @@
-from typing import TYPE_CHECKING, Any, Optional, TypedDict
+from __future__ import annotations
```

Comment:
What is this needed for?

Reply:
it ensures that annotations are evaluated lazily! so you can do a conditional import but reference the type in annotations without having to quote it as a string

in this case it makes it possible to do `Submission | None` even with circular imports, instead of the more verbose `Optional["Submission"]`, since `"Submission" | None` is interpreted as an expression rather than a type, causing it to raise a `TypeError`

It's essentially saying "use the type annotation behaviour of future Python versions even though I'm still on an older version"

Reply:
Ohh!